.github/workflows/build-test.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         python-version: [3.7]
```

Comment:
I wonder if it'd be a good time to bump the minimal version of Python to e.g. 3.11?